File: .py/Program/Baju.py
from Aksesoris import Aksesoris

class Baju (Aksesoris):
    __target = ""
    __size = ""
    __merk = ""

    def __init__(self, ID=0, nama="", jenis="", bahan="", warna="", target="", size="", merk="", stok=0, harga=0):
        super().__init__(ID, nama, jenis, bahan, warna, stok, harga)
        self.__target = target
        self.__size = size
        self.__merk = merk
    
    def setTarget(self, target):
        self.__target = target

    def getTarget(self):
        return self.__target
    
    def setSize(self, size):
        self.__size = size

    def getSize(self):
        return self.__size
    
    def setMerk(self, merk):
        self.__merk = merk

    def getMerk(self):
        return self.__merk

File: .php/Program/Baju.php
<?php
require_once 'Aksesoris.php';

class Baju extends Aksesoris {
    private $target = '';
    private $size = '';
    private $merk = '';

    public function __construct($ID = 0, $nama = '', $jenis = '', $bahan = '', $warna = '', $target = '', $size = '', $merk = '', $stok = 0, $harga = 0, $foto = '') {
        parent::__construct($ID, $nama, $stok, $harga, $foto, $jenis, $bahan, $warna);
        $this->target = $target;
        $this->size = $size;
        $this->merk = $merk;
    }

    public function setTarget ($target) { $this->target = $target; }
    public function getTarget () { return $this->target; }

    public function setSize ($size) { $this->size = $size; }
    public function getSize () { return $this->size; }

    public function setMerk ($merk) { $this->merk = $merk; }
    public function getMerk () { return $this->merk; }
}
?>